Project15-Animations/ViewController.swift

```diff
@@ -8,12 +8,62 @@
 import UIKit
 
 class ViewController: UIViewController {
+    var imageView: UIImageView?
+    var currentAnimation = 0
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        imageView = UIImageView(image: UIImage(named: "penguin"))
+        guard let imageView = imageView else { return }
+        
+        #warning("How can I center this? So it would fit all sizes of screens.")
+        imageView.center = CGPoint(x: 512, y: 384)
```

Comment:
You can center this view using the view controller's view frame as reference as so:

```suggestion
        imageView.center = CGPoint(x:view.center/2, y: view.center/2)
```

This implementation is ok as an exercise, but please keep in mind that there is a more modern way to do it. The `center` property is part of the Coordinate System (`frame`, `bounds` etc.), which we kind of stop using directly after Auto Layout became available. It'd would be better if we had constraints to determine position and size of the views. Please note that constraints can be animated too, here is a post about it: https://useyourloaf.com/blog/animating-autolayout-constraints/.

Reply:
Oh. I was wondering how to do that. 
Yup, much better using the center property.
I will change it right away! 
Thank you!

Reply:
Also, I will save this link for later practice. Interesting how flexible constraints might be.